config/routes.rb

```diff
@@ -296,7 +296,11 @@
         :via => :get
   ####
 
+  #### PublicBodyChangeRequest controller
   resource :change_request, :only => [:new, :create], :controller => 'public_body_change_requests'
+  match 'change_request/new/:body' => 'public_body_change_requests#new',
+        :as => :new_change_request_body,
+        :via => :get
```

Comment:
That doesn't look right to me - rather than adding a single new route for `change_request/new/:body` it's done this:

```diff
-     change_request POST               /change_request(.:format)                                               public_body_change_requests#create
-     new_change_request GET            /change_request/new(.:format)                                           public_body_change_requests#new
+     public_body_change_requests POST  /public_bodies/:public_body_id/change_requests(.:format)                public_body_change_requests#create
+  new_public_body_change_request GET   /public_bodies/:public_body_id/change_requests/new(.:format)            public_body_change_requests#new